site/docs/reference/Connectors/capture-connectors/amazon-s3.md

```diff
@@ -138,14 +133,43 @@ Options are:
    For now, use a prefix in the endpoint configuration to limit the scope of each capture to data of a single file type.
    :::
 
-Only CSV data requires further configuration. When capturing CSV data, you must specify:
+#### CSV configuration
+
+Only CSV data allows further configuration. When capturing CSV data, you may additionally specify:
+
+* **Delimiter**. Options are:
+  * Comma (`","`)
+  * Pipe (`"|"`)
+  * Space (`"0x20"`)
+  * Semicolon (`";"`)
+  * Tab (`"0x09"`)
+  * Vertical tab (`"0x0B"`)
+  * Unit separator (`"0x1F"`)
+  * SOH (`"0x01"`)
+  * Auto
 
-* **Delimiter**
 * **Encoding** type, specified by its [WHATWG label](https://encoding.spec.whatwg.org/#names-and-labels).
+
 * Optionally, an **Error threshold**, as an acceptable percentage of errors.
-* **Escape characters**
+
+* **Escape characters**. Options are:
+  * Backslash (`"\\"`)
+  * Disable escapes (`""`)
+  * Auto
+
 * Optionally, a list of column **Headers**, if not already included in the first row of the CSV file.
```

Comment:
I'll suggest adding:
If any `headers` are provided, then it is assumed that the provided list of `headers` is complete and authoritative. The first row of your CSV file will be assumed to data (not headers), and you must provide a header value for every column in the file.

This is another one that's perhaps a little esoteric (though much less so than 'error threshold'), but it'll be important for some people and they'll likely need a little more guidance on how to use it.